src/util/KVStore/AMOStore.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Jack Khuu, Justin Johnson, Lukas Joswiak
+
+#include "AMOStore.hpp"
+
+#include <stdlib.h>
+
+#include <map>
+#include <string>
+#include <iostream>
+
+#include "Action.hpp"
+#include "AMOCommand.hpp"
+#include "AMOResponse.hpp"
+#include "KVStore.hpp"
+
+namespace AMOStore {
+using std::string;
+using std::map;
+using std::cout;
+using std::endl;
```

Comment:
Don't use using.

Reply:
Not in a header so this is not bad style